addons/test_discuss_full/tests/test_performance.py

```diff
@@ -73,6 +69,10 @@ def setUp(self):
             'employee_id': employee.id,
             'holiday_status_id': self.leave_type.id,
         } for employee in self.employees])
+        self.shortcodes = self.env['mail.shortcode'].with_user(self.users[0]).create([
+            {'source': 'hello', 'substitution': 'Hello, how may I help you?'},
+            {'source': 'bye', 'substitution': 'Thanks for your feedback. Goodbye!'},
+        ])
```

Comment:
I've removed them from init messaging, I think they are no longer tested and can be removed from here for now.